changelogs/fragments/1078-postgresql_ext_fix_version_selection_when_version_is_latest.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- postgresql_ext - fix version selection when ``version=latest``
```

```suggestion
- postgresql_ext - fix version selection when ``version=latest`` (https://github.com/ansible-collections/community.general/pull/1078).
```
when we don't have a separate issue, we put a reference to a pr inself